src/main/java/de/komoot/photon/RequestHandler.java

```diff
@@ -59,10 +59,13 @@ public String handle(Request request, Response response) {
 			limit = 15;
 		}
 
-		List<JSONObject> results = searcher.search(query, lang, lon, lat, limit, true);
+        String tagKey = request.queryParams("tagKey");
+        String tagValue = request.queryParams("tagValue");
```

Comment:
ok, this is a difficult issue as there is no real convention for url parameters. camelcase should be used in java application. However when it comes to urls or query parameters i would rather use underscore variants: `tag_key` & `tag_value` identical as it was defined in our [mapping.json](https://github.com/komoot/photon/blob/master/es/mappings.json#L25)

But as I said, there is no real convention, see
 - http://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris
 - http://stackoverflow.com/questions/778203/are-there-any-naming-convention-guidelines-for-rest-apis